### DISCUSSION TITLE
Multitask learning using Detectron2

### DISCUSSION THREAD

**Discussion by Dheeru66k** [facebookresearch/detectron2#4433] [Ideas]
Hello Team !! I am trying to do Multi task learning , i want Semantic, Depth estimation and Obj detection to be performed by a single network , any suggestions how can i do it using detectron2. i found similar one like "Panoptic FPN" which is performing semantic and obj detection tasks together. But i am not have an idea how can i regiter a new model where it can also perform Depth estimation.

**Comment by saltoricristiano** [comment]
Hi @Dheeru66k, 

did u manage to implement depth estimation with detectron2 in the end?
Thanks!
